hermes/probe/create_file.go

```diff
@@ -0,0 +1,116 @@
+package probe
+<<<<<<< HEAD
+
+import (
+	"context"
+	"crypto/sha1"
+	"fmt"
+	"io"
+	"math/rand"
+	"os"
+	"sync"
+	"time"
+
+	"cloud.google.com/go/storage"
+	"github.com/google/cloudprober/logger"
+	"github.com/google/cloudprober/metrics"
+	"github.com/google/cloudprober/probes/options"
+	"github.com/google/cloudprober/targets/endpoint"
+	"github.com/googleinterns/step224-2020/blob/evan/hermesMain/cmd"
+)
+
+type RandomFile struct {
+	Seed int64
+	Size int // File size in bytes
+}
+
+func (f *RandomFile) NewReader() *randomFileReader {
+	return &randomFileReader{size: f.Size, rand: rand.New(rand.NewSource(f.Seed))}
+}
+
+type randomFileReader struct {
+	size int
+	// Current reading index.
+	i    int
```

Comment:
Could you change this to `idx` or something slightly more descriptive, as this is very terse.

Reply:
It's OK to have short variable names for such small-scoped variables (it's only used in the next ~10 lines).
Also, `i` is generally used as the index variable.
The standard library can be used as a reference:
https://golang.org/src/strings/reader.go